custom_components/teslafi/coordinator.py

```diff
@@ -91,6 +96,11 @@ async def _refresh(self) -> TeslaFiVehicle:
 
         self._vehicle.update_non_empty(current)
         LOGGER.debug("Remote data last updated %s", self._vehicle.last_remote_update)
+        
+        api_requests = await self._client.request_counts()
```

Comment:
Since we get the latest counts in response to a command, and not on every refresh of data, we don't need these added lines.